### PR TITLE
Change participant service to conform to change in db

### PR DESCRIPTION
**Pull request by boompttn**

### CODE REVIEW

File: proto/hts/participant/service.proto [hunk 1]
```diff
@@ -57,6 +58,23 @@ message GetUpcomingEventsRequest {
...
 
+message GetEventsByTagIdRequest {
+    int64 tag_id = 1;
+}
+message GetEventsByOrganizationIdRequest {
+    int64 organization_id = 1;
+}
+
+message GetEventsByFacilityIdRequest {
+    int64 facility_id = 1;
```
Comment (ChanyutJean) [inline proto/hts/participant/service.proto:68]: can merge all 3 messages to int64 id = 1

**Review by ChanyutJean — COMMENTED** [review]
Checks are failing.

File: proto/hts/participant/service.proto (hunk 1 above)
Comment (boompttn) [reply proto/hts/participant/service.proto:68]: what do you mean?

**Comment by boompttn** [comment]
> Checks are failing.

image
It fails in Java service

File: proto/hts/participant/service.proto (hunk 1 above)
Comment (ChanyutJean) [reply proto/hts/participant/service.proto:68]: GetByFacility, GetByEvent, ... have same type of input and can be generalized into one message

**Pull request merged**